command/acl/bootstrap/bootstrap.go

```diff
@@ -43,13 +46,46 @@ func (c *cmd) Run(args []string) int {
 		return 1
 	}
 
+	args = c.flags.Args()
+	if l := len(args); l < 0 || l > 1 {
+		c.UI.Error("This command takes up to one argument")
+		return 1
+	}
+
+	var terminalToken []byte
+	var err error
+
+	if len(args) == 1 {
+		switch args[0] {
+		case "":
+			terminalToken = []byte{}
+		case "-":
+			terminalToken, err = ioutil.ReadAll(os.Stdin)
+		default:
+			file := args[0]
+			terminalToken, err = ioutil.ReadFile(file)
+		}
```

Comment:
There is already a [CLI helper](https://github.com/hashicorp/consul/blob/main/command/helpers/helpers.go#L57) to assist with this sequence of operations that we should use:

```suggestion
		terminalToken, err = helpers.LoadDataSourceNoRaw(args[0], os.Stdin)
```
Note that this is going to return a `string` already, so the typing for `terminalToken` needs fixing too. You could probably parse it directly into `var initialManagementToken string`.